abdiff/core/run_ab_transforms.py

```diff
@@ -1 +1,203 @@
 """abdiff.core.run_ab_transforms"""
+
+import logging
+import os
+import time
+from datetime import timedelta
+from pathlib import Path
+from time import perf_counter
+
+import docker
+from docker.models.containers import Container
+
+from abdiff.config import Config, configure_logger
+from abdiff.core.exceptions import DockerContainersNotFoundError
+from abdiff.core.utils import read_run_json, update_or_create_run_json
+
+CONFIG = Config()
+logger = logging.getLogger(__name__)
+configure_logger(logger=logger, verbose=False)
+
+# init Docker client
+DockerClient = docker.from_env()
+
+
+def run_ab_transforms(
+    run_directory: str,
+    image_name_a: str,
+    image_name_b: str,
+    input_files: list[str],
+) -> tuple[list[str], ...]:
+    """Run Docker containers with versioned images of Transmogrifier."""
+    start_time = perf_counter()
+
+    # get run ID (used to label containers)
+    run_id = str(run_directory).split("/")[-1]
+
+    # create directories for 'transformed' outputs
+    transformed_directory_a, transformed_directory_b = create_directories(run_directory)
+
+    # create list to track container IDs for a given 'run'
+    container_ids = []
```

Comment:
Very minor: I'd kind of expect this right above the loop that is going to utilize it (around line  # 51)

Reply:
Agreed!

Reply:
Good idea -- and will remember this suggestion moving forward! Updated.